slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/interchange/InterchangeV5Exporter.java

```diff
@@ -225,4 +257,28 @@ private void writeAttestations(
     }
     jsonGenerator.writeEndArray();
   }
+
+  public class IncrementalInterchangeV5Exporter implements IncrementalExporter {
+    final JsonGenerator jsonGenerator;
+
+    public IncrementalInterchangeV5Exporter(final OutputStream outputStream) throws IOException {
+      jsonGenerator = mapper.getFactory().createGenerator(outputStream);
+      startInterchangeExport(jsonGenerator);
```

Comment:
I like how the encapsulation of this class allows you to reuse this method as well as keeping the class well scoped.